app/config/index.js

```diff
@@ -142,6 +142,18 @@ function argv(configPath, appVersion) {
 				describe: 'Custom Client Certs password for corporate authentication (certificate must be in pkcs12 format)',
 				type: 'string'
 			},
+
+			ssoUser: {
```

Comment:
Could you update the 'REAME.md' file in this folder with this two options? I try to put them in alphabetical order but people tend to ignore that 😄